app/src/services/section.services.ts

```diff
@@ -26,10 +26,40 @@ export const saveSection = async (data: any, id: any/*, token: string*/) => {
     );
 };
 
+
+/**
+ * Create a new section for a course
+ * 
+ * @param data Input data, should be a Section interface
+ * @param id id of the section
+ * @param token token of the user, currently ignored
+ * @returns respons from the backend post request
+ */
+const createSection = async (data: any, id: any, token: string) => {
+    return await axios.post(
+      `${BACKEND_URL}/api/sections/create/${id}`,
```

Comment:
Do not have actions in endpoint

should be api/sections/:id

Should it be PUT instead of POST?

Reply:
This has now been changed to be a put